package.json

```diff
@@ -12,11 +12,15 @@
   ],
   "scripts": {
     "build": "webpack -p",
+    "heroku-postbuild": "make emojis && webpack -p --config ./webpack.dev.js",
+    "start": "webpack-dev-server --config ./webpack.dev.js --open",
     "lint": "eslint . --ext .jsx --ext .js",
     "pretest": "npm run lint && npm run prettier",
     "prettier": "prettier --list-different --write \"./**/**.{js,jsx}\"",
-    "test": "jest --coverage --runInBand",
+    "test": "jest --coverage --runInBand --testPathIgnorePatterns '__tests__/browser'",
     "test.watch": "jest --watch --coverage=false",
+    "test.browser": "jest --runInBand --testPathPattern '__tests__/browser'",
+    "test.browser.watch": "npm run test.browser -- --watch",
```

Comment:
We can probably just get rid of this script in favor of executing `npm run test.browser -- --watch` directly!
```suggestion
```